src/utility/index.jsx

```diff
@@ -39,27 +40,34 @@ export function abbreviateNumber(num) {
   } else if (num >= 1000000000000) {
     return `${Number((num / 1000000000000).toFixed(1))}${strings.abbr_trillion}`;
   }
+
   return num.toFixed(0);
-}
+};
+
 export function formatSeconds(input) {
   if (!isNaN(parseFloat(input)) && isFinite(input)) {
     const absTime = Math.abs(input);
     const minutes = Math.floor(absTime / 60);
     const seconds = pad(Math.floor(absTime % 60), 2);
+
     let time = ((input < 0) ? '-' : '');
     time += `${minutes}:${seconds}`;
+
```

Comment:
I'm not a fan of all the space this creates (adding a space before returns). Doesn't really seem necessary to me